garrysmod/lua/vgui/dhtml.lua

```diff
@@ -53,8 +53,8 @@ function PANEL:QueueJavascript( js )
 	end
 
 	self.JS = self.JS or {}
-
-	table.insert( self.JS, js )
+	self.JS[self.JS + 1] = js

```

Comment:
```suggestion
	self.JS[#self.JS + 1] = js
```

Just making https://github.com/Facepunch/garrysmod/pull/1709#issuecomment-674363812 more obvious